### ISSUE TITLE
Interface Functions Not Exist Warning When Included In XML File

### ISSUE THREAD

**Issue by triwav** [rokucommunity/brighterscript#523]
Roku's RALE includes Interface Functions that point to code in the xml file. brighterscript complains that these don't exist.

**Comment by TwitchBronBron** [comment]
Yeah...we don't currently parse brightscript code located in the xml files. It's on the list, but way down.  PR #408 will hopefully lay the groundwork for some of this in v1.
